tests/test_start_date.py

```diff
@@ -114,17 +136,28 @@ def test_run(self):
                 start_date_key_sync_1 = set(start_date_keys_list_1)
                 start_date_key_sync_2 = set(start_date_keys_list_2)
 
-                # Verify bookmark key values are greater than or equal to start date of sync 1
-                for start_date_key_value in start_date_key_sync_1:
-                    self.assertGreaterEqual(self.dt_to_ts(start_date_key_value), start_date_1_epoch)
+                if expected_replication_method == self.INCREMENTAL:
+
+                    # Verify bookmark key values are greater than or equal to start_date of sync 1
+                    for start_date_key_value in start_date_key_sync_1:
+                        self.assertGreaterEqual(self.dt_to_ts(start_date_key_value), start_date_1_epoch)
```

Comment:
Update `dt_to_ts` function in a base.py to convert date as per provided format. Add additional argument in function to define the format.

Reply:
Updated the dt_to_ts function to call with date-format.